wire-compiler/src/main/java/com/squareup/wire/WireCompiler.java

```diff
@@ -221,32 +128,46 @@ public static void main(String... args) throws Exception {
     return null;
   }
 
-  public WireCompiler(String protoPath, List<String> sourceFileNames, List<String> roots,
-      String outputDirectory, String registryClass, boolean emitOptions, List<String> enumOptions,
+  @Deprecated
+  public WireCompiler(String protoPath, String outputDirectory, List<String> sourceFileNames,
```

Comment:
The parameters to this method were re-ordered which is a backwards-incompatible public API change and will break people's (including all our Gradle-based Android projects') code.